Prettier.sublime-settings

```diff
@@ -1,4 +1,5 @@
 {
+  "extensions": ["js", "jsx"],
```

Comment:
I wasn't sure what name was the best for this, `extensions` sounds good, maybe `extensions_whitelist`?